src/index.html

```diff
@@ -11,7 +11,220 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      method="POST"
+      onsubmit="onSubmit()"
+    >
```

Comment:
```suggestion
    <form method="POST" onsubmit="onSubmit()">
``` 
2 or fewer attributes can be written in 1 line.